src/js/components/PodInfo.js

```diff
@@ -0,0 +1,160 @@
+import classNames from 'classnames';
+import {Dropdown} from 'reactjs-components';
+import React from 'react';
+
+import HealthBar from './HealthBar';
+import PageHeader from './PageHeader';
+import Pod from '../structs/Pod';
+import ServiceActionItem from '../constants/ServiceActionItem';
+import StatusMapping from '../constants/StatusMapping';
+import StringUtil from '../utils/StringUtil';
+
+class PodInfo extends React.Component {
+  constructor() {
+    super();
+  }
+
+  getActionButtons() {
+    const {pod} = this.props;
+
+    const dropdownItems = [{
+      className: 'hidden',
+      id: ServiceActionItem.MORE,
+      html: '',
+      selectedHtml: 'More'
+    }, {
+      className: classNames({
+        hidden: pod.getInstancesCount() === 0
+      }),
+      id: ServiceActionItem.RESTART,
+      html: 'Restart'
+    }, {
+      className: classNames({
+        hidden: pod.getInstancesCount() === 0
+      }),
+      id: ServiceActionItem.SUSPEND,
+      html: 'Suspend'
+    }, {
+      id: ServiceActionItem.DESTROY,
+      html: <span className="text-danger">Destroy</span>
+    }];
+
+    let actionButtons = [
+      <button className="button flush-bottom button-stroke button-inverse"
+        key="action-button-edit"
+        onClick={() =>
+          this.props.onActionsItemSelection({id: ServiceActionItem.EDIT})}>
+        Edit
+      </button>,
+      <Dropdown
+        key="actions-dropdown"
+        anchorRight={true}
+        buttonClassName="button button-stroke button-inverse dropdown-toggle"
+        dropdownMenuClassName="dropdown-menu"
+        dropdownMenuListClassName="dropdown-menu-list"
+        dropdownMenuListItemClassName="clickable"
+        wrapperClassName="dropdown flush-bottom"
+        items={dropdownItems}
+        persistentID={ServiceActionItem.MORE}
+        onItemSelection={this.props.onActionsItemSelection}
+        scrollContainer=".gm-scroll-view"
+        scrollContainerParentSelector=".gm-prevented"
```

Comment:
Let's not add these classes!
If you don't want Gemini then use the `useGemini` option. But why would you want to disable it?